test/celeritas/global/StepperGeant.test.cc

```diff
@@ -800,7 +800,6 @@ TEST_F(LeadBox, host)
     // tracking cut
     EXPECT_EQ(1, result.calc_avg_steps_per_primary());
     EXPECT_EQ(1, result.num_step_iters());
-    EXPECT_SOFT_EQ(1, result.calc_avg_steps_per_primary());
```

Comment:
Was this intentional? Not sure how this would be affected...

Reply:
Yeah, it doesn't have anything to do with this PR, it was just a duplicate check I'd left in by mistake. I can revert that and clean it up later if you'd prefer.

Reply:
Ah I see it now. Works for me!